### PR TITLE
Fix bug with plantuml-filename in a directory

### PR DESCRIPTION
**Pull request by Felixoid**
With the latest version, it's impossible to use files outside of CWD for plantuml-filename.

I'm very open to another function name since not glad about the current one.

### CODE REVIEW

**Comment by Felixoid** [comment]
Good morning @timofurrer. Do you have objections against this?

**Comment by Felixoid** [comment]
Hey @timofurrer, I wish you a happy coming New Year.

Do you mind to merge it?

File: pandoc_plantuml_filter.py [hunk 1]
```diff
@@ -16,6 +16,19 @@
...
+    dest_dir = os.path.dirname(dest)
+    if not os.path.exists(dest_dir):
+        os.makedirs(dest_dir)
+
+    if os.path.exists(dest):
+        os.remove(dest)
+
+    src = os.path.relpath(src, dest_dir)
+    os.symlink(src, dest)
+    pass
```
Comment (timofurrer) [inline pandoc_plantuml_filter.py]:
```suggestion
```

**Review by timofurrer — CHANGES_REQUESTED** [review]
I'll merge, once the `pass` statement is removed 🎉 

Thanks!

File: pandoc_plantuml_filter.py (hunk 1 above)
Comment (Felixoid) [reply pandoc_plantuml_filter.py]: Thank you, cleaned out

**Comment by Felixoid** [comment]
Thank you very much! Could you be so nice and release a new version to make me able to update the package in AUR? https://aur.archlinux.org/packages/pandoc-plantuml-filter-py/

**Pull request merged**